src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/softdelete/SoftDeleteService.kt

```diff
@@ -62,7 +65,7 @@ class SoftDeleteService(
     val licencesRecords = licencesToMigrate(initialIdToProcess, numberToMigrate)
     val lastIdProcessed = licencesRecords.content.lastOrNull()?.first?.id
     log.info("Last Id processed in batch: ${lastIdProcessed ?: " no records processed"}")
-    val licencesToSoftDelete = applyAnySoftDeletes(licencesRecords)
+    val licencesToSoftDelete = applyAnySoftDeletes(licencesRecords, "MIGRATION")
```

Comment:
I'm not the best Kotlin-er, but I believe Andy has said to me previously that if there's a list of constant string values, it's best to put them in an enum and reference the enum.
eg CVL [AuditEventTypes](https://github.com/ministryofjustice/create-and-vary-a-licence-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/AuditEventType.kt) which is referenced like [this](https://github.com/ministryofjustice/create-and-vary-a-licence-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/DeactivateLicencesService.kt#L64)